src/libraries/MinimalDelegationStorage.sol

```diff
@@ -6,6 +6,7 @@ import {EnumerableSetLib} from "solady/utils/EnumerableSetLib.sol";
 struct MinimalDelegationStorage {
     EnumerableSetLib.Bytes32Set keyHashes;
     mapping(bytes32 keyHash => bytes encodedKey) keyStorage;
+    mapping(uint192 key => uint256 nonce) nonceSequenceNumber;
```

Comment:
techinically this could be uint192 => uint64 for key => seq right?

Reply:
yeah but you can also just define it as a 

uint256 => uint256 so that you don't have to downcast all the time 

which is what I would recommend instead of passing around uint192/uint64s all the time and just document in the interface how a uint256 nonce value is read

Reply:
okay! was following what they did in entrypoint, but will switch it to this way

Reply:
yeah it does need to include the key im just saying use the types uint256 => uint256